src/SWP/Bundle/CoreBundle/Security/Provider/ExternalOauthResourceOwner.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace SWP\Bundle\CoreBundle\Security\Provider;
+
+use League\OAuth2\Client\Provider\GenericResourceOwner;
+use League\OAuth2\Client\Tool\ArrayAccessorTrait;
+
+class ExternalOauthResourceOwner extends GenericResourceOwner
+{
+    use ArrayAccessorTrait;
+
+    protected $response;
+
+    /**
+     * {@inheritdoc}
+     */
```

Comment:
we doin't need them (inheritdoc) anymore from php 7.1 ;) We are cleaning that in code base also when we touch some files.